code/modules/spacepods/equipment.dm

```diff
@@ -0,0 +1,354 @@
+	shot_cost = 1200
+	shots_per = 3
+	fire_sound = 'sound/weapons/taser2.ogg'
+	fire_delay = 30
+	overlay_icon = "pod_weapon_disabler"
+
+/obj/item/device/spacepod_equipment/weaponry/laser
+	name = "laser system"
+	desc = "A weak laser system for space pods, fires concentrated bursts of energy."
+	icon_state = "weapon_laser"
+	projectile_type = /obj/item/projectile/beam
+	shot_cost = 600
+	fire_sound = 'sound/weapons/Laser.ogg'
+	overlay_icon = "pod_weapon_laser"
+
+// MINING LASERS
+/obj/item/device/spacepod_equipment/weaponry/basic_pod_ka
+	name = "weak kinetic accelerator"
+	desc = "A weak kinetic accelerator for space pods, fires bursts of energy that cut through rock."
+	icon = 'goon/icons/pods/ship.dmi'
+	icon_state = "pod_taser"
+	projectile_type = /obj/item/projectile/kinetic/pod
+	shot_cost = 300
+	fire_delay = 14
+	fire_sound = 'sound/weapons/Kenetic_accel.ogg'
+
+/obj/item/device/spacepod_equipment/weaponry/pod_ka
+	name = "kinetic accelerator system"
+	desc = "A kinetic accelerator system for space pods, fires bursts of energy that cut through rock."
+	icon = 'goon/icons/pods/ship.dmi'
+	icon_state = "pod_m_laser"
+	projectile_type = /obj/item/projectile/kinetic/pod/regular
+	shot_cost = 250
+	fire_delay = 10
+	fire_sound = 'sound/weapons/Kenetic_accel.ogg'
+
+/obj/item/device/spacepod_equipment/weaponry/plasma_cutter
+	name = "plasma cutter system"
+	desc = "A plasma cutter system for space pods. It is capable of expelling concentrated plasma bursts to mine or cut off xeno limbs!"
+	icon = 'goon/icons/pods/ship.dmi'
+	icon_state = "pod_p_cutter"
+	projectile_type = /obj/item/projectile/plasma
+	shot_cost = 250
+	fire_delay = 10
+	fire_sound = 'sound/weapons/plasma_cutter.ogg'
+	overlay_icon = "pod_weapon_plasma"
+
+/obj/item/device/spacepod_equipment/weaponry/plasma_cutter/adv
+	name = "enhanced plasma cutter system"
+	desc = "An enhanced plasma cutter system for space pods. It is capable of expelling concentrated plasma bursts to mine or cut off xeno faces!"
+	icon_state = "pod_ap_cutter"
+	projectile_type = /obj/item/projectile/plasma/adv
+	shot_cost = 200
+	fire_delay = 8
+
+/*
+///////////////////////////////////////
+/////////Misc. System///////////////////
+///////////////////////////////////////
+*/
+
+/obj/item/device/spacepod_equipment/misc
+	name = "pod misc"
+	desc = "You shouldn't be seeing this"
+	icon = 'goon/icons/pods/ship.dmi'
+	icon_state = "blank"
+	var/enabled
+
+/obj/item/device/spacepod_equipment/misc/tracker
+	name = "\improper spacepod tracking system"
+	desc = "A tracking device for spacepods."
+	icon_state = "pod_locator"
+	enabled = TRUE
+
+/obj/item/device/spacepod_equipment/misc/tracker/attackby(obj/item/I as obj, mob/user as mob, params)
+	if(istype(I, /obj/item/screwdriver))
+		if(enabled)
+			enabled = FALSE
+			user.show_message("<span class='notice'>You disable \the [src]'s power.")
+			return
+		enabled = TRUE
+		user.show_message("<span class='notice'>You enable \the [src]'s power.</span>")
```

Comment:
to_chat
edit: actually i'm not sure which is better but I'm pretty sure to_chat is the same thing.